src/agents/customer_group.jl

```diff
@@ -383,9 +383,10 @@ function CustomerGroup(input_filename::AbstractString, model_data::HEMData; id =
         ),
     )
 
-    function generate_coefficients(index_h, h_to_sector, coefficients)
+    function generate_coefficients(index_h, index_h_to_sector_map, coefficients)
+        map_dict = Dict(index_h_to_sector_map.set)
```

Comment:
```suggestion
        map_dict = Dict(index_h_to_sector_map)
```

based on line 1565 in regulator.jl, e.g., where it says:
```julia
        # Assign rates to customers
        h_to_sector = Dict(model_data.index_h_sector_map)
```